### DISCUSSION TITLE
.dep repository pack 455MB

### DISCUSSION THREAD

**Discussion by jasperf** [deployphp/deployer#3175] [Help needed]
Was cleaning up the hard disk and came across Deployer repository package worth 455MB:

```
du -sh /home/forge/staging.site.com/.dep/repo/objects/pack/pack-321e251bdd7055872111a5dfb75f46ee0b94f67f.pack
455M    /home/forge/staging.site.com/.dep/repo/objects/pack/pack-321e251bdd7055872111a5dfb75f46ee0b94f67f.pack
```

is this a normal size? Pretty large...

**Comment by antonmedv** [comment]
Yes, it’s your repo what Deployer clones on disk for faster updates.

**Reply by jasperf** [reply]
Oh I see. Did not know that. Thanks @antonmedv
